src/awesomeSezzle.js

```diff
@@ -1118,7 +1121,56 @@ class AwesomeSezzle {
 				}
 			} else if (this.altModalHTML) {
         modalNode.innerHTML = this.altModalHTML;
-      } else {
+      } else if(this.modalTheme==="grayscale")
```

Comment:
This works, but the file already exceed recommendations. Per mine and Hardik's earlier comment, there is a better way to write this. We could use the default HTML lower down and use templates to modify the needed class-names. For example: <div class="sezzle-modal${this.modalTheme==="grayscale" && "-grayscale"}">

Reply:
Also be sure to run `npx webpack` and test in playground/index.html to confirm it is working properly.